test/functional/discovered_hosts_controller_test.rb

```diff
@@ -92,7 +92,7 @@ def test_reboot_error
     ::ProxyAPI::BMC.any_instance.expects(:power).raises("request must fail")
     post "reboot", { :id => host.id }, set_session_user
     assert_redirected_to discovered_hosts_url
-    assert_equal "Failed to reboot host #{host.name} with error request must fail", flash[:error]
+    assert_equal "Failed to reboot host #{host.name}", flash[:error]
```

Comment:
This isn't a test for orchestration though, it's a test for the UI, which is now worse than it was.